x-pack/plugins/lens/public/datatable_visualization/components/cell_value.tsx

```diff
@@ -74,7 +76,10 @@ export const createGridCell = (
          */
         dangerouslySetInnerHTML={{ __html: content }} // eslint-disable-line react/no-danger
         data-test-subj="lnsTableCellContent"
-        className={`lnsTableCell--${currentAlignment}`}
+        className={classNames({
```

Comment:
What if we remove `x-pack/plugins/lens/public/datatable_visualization/components/table_basic.scss` in favor of using the `emotion/css` ? 

I see we just need to set `text-align`, `white-space` and `height` styles

Reply:
Makes sense, I would like to split it out into a separate PR though - I will add it to our tech debt list